package.json

```diff
@@ -33,10 +33,10 @@
 		"ci:test:mocha:coverage": "c8 --no-clean npm run test:mocha:report",
 		"ci:test:realsvc:local": "pnpm run -r --no-sort --stream --no-bail test:realsvc:local:report",
 		"ci:test:realsvc:local:coverage": "c8 --no-clean pnpm recursive --no-sort --stream --no-bail run test:realsvc:local:report",
-		"ci:test:realsvc:local:full": "pnpm run -r --no-sort --stream --no-bail test:realsvc:local:report:full",
+		"ci:test:realsvc:local:full:coverage": "c8 --no-clean pnpm recursive --no-sort --stream --no-bail run test:realsvc:local:report:full",
```

Comment:
Was there a reason for `pnpm recursive ... run` instead of ` pnpm run -r ...` like before (or in the other changed line)?

Reply:
It's so it's like the other `local:coverage` command